src/algebra/group_with_zero/basic.lean

```diff
@@ -366,7 +366,7 @@ classical.by_contradiction $ λ ha, h₁ $ mul_right_cancel' ha $ h₂.symm ▸
 end cancel_monoid_with_zero
 
 section group_with_zero
-variables [group_with_zero G₀]
+variables [group_with_zero G₀] {a b c g h x : G₀}
```

Comment:
I'm not convinced this actually makes things better (unless you also renamed the x g and h to a b c), but also realistically it doesn't matter at all.

Reply:
I personally find lemmas where implicit arguments like this are omitted more readable (and I didn't want to add them to the lemmas I just wrote)

Reply:
Maybe I've just been bitten too much by variables at the top of the file that steal the names I want in a smaller section and have typeclass assumptions I can't revert, and am overgeneralizing.

No change needed, the d+ was just for CI